Feature/blog posts

mercurrie opened the pull request:
completed feature blog posts to website deployed in flask

mercurrie committed:
break

mercurrie committed:
comlete

completed simple blog post website deployed in flask

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,30 @@
+HOME_PATH = '/'
+POST_ID_KEY = "id"
+SHOW_POST_PATH = "/path/<int:index>"
+blog_response = requests.get(BLOG_API)
```
You'd probably want to call this from within the function.  Otherwise, it won't change results if something gets added because this is invoked at the beginning of the file.

mercurrie replied:
I created a new function to return the get request, and called it within the route functions.

michaelhousman reviewed (commented):
This is good.  Left one comment.

mercurrie committed:
refactor

included function to handle get requests.
called function as needed within route functions

mercurrie committed:
type cast

included type cast to functions with  parameters

mercurrie committed:
Blog Upgrade

Utilized Flask, Bootstrap, and jinja top upgrade the design of the blog site

mercurrie committed:
picture of site
